spec/features/two_factor_authentication/sign_in_spec.rb

```diff
@@ -183,9 +183,16 @@ def submit_prefilled_otp_code
       sign_in_before_2fa(user)
 
       code = RecoveryCodeGenerator.new(user).create
+      code_words = code.split(' ')
       click_link t('devise.two_factor_authentication.recovery_code_fallback.link')
-      fill_in 'code', with: code
-      click_button t('forms.buttons.submit.default')
+
+      fields = page.all('input[type="text"]')
+
+      fields.size.times do |index|
```

Comment:
Looks like we repeat this split-and-set-words procedure in a few tests, how about an `enter_recovery_code` helper?

I think it could handle splitting and sutff like:

```ruby
def enter_recovery_code(recovery_code, selector: 'input[type="text"]')
  code_words = code.split(' ')
  fields = page.all(selector)

  fields.zip(code_words) do |field, word|
    field.set(word)
  end
end
```